### ISSUE TITLE
autocompletion when \connect -ing to different databases

### ISSUE THREAD

**Issue by rafaelosoto** [amjith/pgcli#174]
Hi Guys, 

This is my first time opening an issue on gh so forgive my faults if any. I looked for open tickets pertaining to this but couldn't find any.

Whenever I connect to a new db using \connect my autocompletion for tables remains the same as the previous db table I was on.

-rafe

**Comment by amjith** [comment]
Could you tell me how you installed pgcli? 

If it was installed via `brew` there was an old bug which was fixed in the newer versions. 

You can try installing the latest version using `pip install pgcli`. 

There are detailed instructions on how to install it using pip here: http://pgcli.com/install

I'll update the brew version to the latest when I get a chance.

**Comment by amjith** [comment]
Nope! You're right it's a bug. 

You can change to a new database using just `\c` which works as expected. But if you try `\connect` then it doesn't work. 

I'll fix it soon. Sorry about that. But in the mean time you can change to a different database using `\c databasename`. 

Thanks for reporting.

**Comment by rafaelosoto** [comment]
Awesome ^-^ just trying to help, really like the software!

**Comment by amjith** [comment]
It is now fixed in the master branch. I'll plan on releasing it soon. 

You can try out the version from master by directly installing pgcli from the git repo using the following command: 

```
pip install -U -e git+https://github.com/amjith/pgcli.git#egg=pgcli
```

Cheers!

**Comment by amjith** [comment]
This is now released in 0.16.2.

http://pgcli.com/v0.16.2.html

You can upgrade using 

```
pip install -U pgcli 
```
